recipe/meta.yaml

```diff
@@ -7,37 +7,59 @@ package:
 source:
   url: https://github.com/scikit-learn/scikit-learn/archive/{{ version }}.tar.gz
   sha256: b7c0a9fb8dfcefcfc7ef8fe02a064d194980251d57efaea972cb76005afb3c63
+  patches:
+   - patches/sklearn-pr24141.patch
+   - patches/sklearn-pr24483-aa2172.patch
+   - patches/sklearn-pr24483-a21429.patch
+   - patches/sklearn-pr24483-937b21.patch
+   - patches/sklearn-pr24483-0b3128.patch
+   - patches/sklearn-pr24543-af3056.patch
+   - patches/sklearn-pr24543-c7363b.patch
+   - patches/sklearn-pr24543-883764.patch
+   - patches/sklearn-pr24543-2861a7.patch
+   - patches/sklearn-pr24543-78a51f.patch
+   - patches/sklearn-pr24543-17035b.patch
 
 build:
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .  --verbose
+  script: python -m pip install . --no-deps --no-build-isolation --ignore-installed -vv
   skip: true  # [py<38]
+  missing_dso_whitelist:
+    - '*/libc++.1.dylib'  # [osx]
+    - '*/libomp.dylib'    # [osx]
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - llvm-openmp  # [osx]
+    - patch # [not win]
+    - m2-patch # [win]
   host:
     - python
     - cython >=0.29.24
-    - joblib >=1.0.0
-    - llvm-openmp  # [osx]
+    - joblib
+    - llvm-openmp 14.0.6  # [osx]
     - numpy   1.19  # [py==38 or py==39]
     - numpy   1.21  # [py==310]
+    - numpy   1.23  # [py>=311]
     - pip
-    - scipy >=1.3.2
+    - scipy >=1.3.2,<=1.9.3 # [py<311]
+    - scipy 1.10.0 # [py>=311]
     - setuptools <60.0
-    - threadpoolctl >=2.0.0
+    - threadpoolctl
     - wheel
   run:
     - python
-    - joblib >=1.0.0
+    - joblib
     - llvm-openmp  # [osx]
     - {{ pin_compatible('numpy') }}
-    - scipy >=1.3.2
-    - threadpoolctl >=2.0.0
+    - scipy >=1.3.2,<=1.9.3 # [py<311]
+    - scipy 1.10.0          # [py>=311]
```

Comment:
Why are there different pinnings here? I mean I can't see it in the upstream, but py 3.11 is also not a target version in the upstream.

Reply:
same comment, why not use scipy 1.9.3 on 3.11? It's also available. Or does it conflict with another package?